client/state/billing-transactions/test/reducer.js

```diff
@@ -235,4 +244,41 @@ describe( 'reducer', () => {
 			expect( state ).to.eql( {} );
 		} );
 	} );
+
+	describe( '#individualTransactions', () => {
```

Comment:
Reducer tests could use some tests for the cases when we already have data for a certain `receiptId` and we want to test whether it overrides it properly - proposed tests only test against an empty state tree.